src/promptflow/tests/sdk_cli_test/unittests/test_utils.py

```diff
@@ -468,3 +472,29 @@ def mock_http_request():
 
         http_retry_wrapper(mock_http_request, tries=2, delay=1, backoff=1)()
         assert counter == 2
+
+    def test_get_mac_address(self):
+        mac_address = None
+        try:
+            import psutil
+
+            net_address = psutil.net_if_addrs()
```

Comment:
what does this test trying to catch?

Reply:
To avoid errors caused by psutil not being installed, but it seems to be ineffective, I removed it.